packages/desktop-ui-lib/src/lib/permission-manager/permission-manager.component.html

```
@@ -26,65 +26,67 @@ <h6>{{ 'TIMER_TRACKER.PERMISSIONS.SCREEN_RECORDING' | translate }}</h6>
-							<button
-								nbButton
-								status="primary"
-								size="small"
-								(click)="openSystemSettings()"
-							>
-								<nb-icon icon="settings-2-outline"></nb-icon>
-								{{ 'TIMER_TRACKER.PERMISSIONS.OPEN_SYSTEM_SETTINGS' | translate }}
-							</button>
+						<div class="action-buttons">
+							@if (isMac()) {
+								<button nbButton status="primary" size="small" (click)="openSystemSettings()">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)